core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -104,6 +115,25 @@ public TabletsMetadata build() {
       }
     }
 
+    private TabletsMetadata buildExtents(AccumuloClient client) {
+
+      try {
+        BatchScanner scanner = client.createBatchScanner(level.metaTable(), Authorizations.EMPTY);
+
+        List<Range> ranges =
+            extents.stream().map(e -> e.toMetaRange()).collect(Collectors.toList());
```

Comment:
3 possible changes here. The first is `KeyExtent::toMetaRange` for the map function. The other two changes are attempts to make this line shorter for readability. Could use `var` here, or static import `toList()`. I did both below, but it's up to you what you're comfortable with. I think it's worth trying to get it to not wrap, though.

```suggestion
        var ranges = extents.stream().map(KeyExtent::toMetaRange).collect(toList());
```